github/src/main/kotlin/gropius/sync/github/Outgoing.kt

```diff
@@ -289,6 +291,28 @@ class Outgoing(
         }
     }
 
+    /**
+     * Mutate an RemovedLabelEvent upto GitHub
+     * @param imsProjectConfig active config
+     * @param issueInfo info of the issue containing the timeline item
+     * @param userList users that have contributed to the event
```

Comment:
does not exist, replace with doc for user?

Reply:
3e4f88b87b2b5fefa5110c93c35b9bea1b3bbd39

Reply:
492363db92e638420d8eb650c9efd33d1c25dfc0